docs/tutorials/custom_segmentation_trainer.ipynb

```diff
@@ -55,7 +55,6 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "6",
```

Comment:
I have no idea why Sphinx doesn't like this line, but it was resulting in warnings in the RtD CI.